USAGE.md

```diff
@@ -7,102 +7,113 @@ This file describes how to run the project and develop against it.
 ## Requirements
 
 - Docker
-- Makefile
+- Make
+- NodeJS 8.0.0
 
-## Getting Started (OSX and LINUX)
+## Getting Started
 
-- Install Docker and Makefile
-- Define .env files in frontend and backend folders
+- Install Requirements listed above
+- On Windows, note the following:
+    - If containers are not working, they may not be enabled, enabling them in docker settings and restarting the machine fixes this
+    - Drive sharing is disabled by default, make sure to share your local drive in docker settings
 
-- Run the following command to build and run the project using Docker.
+## Setting up local development
 
-```
-make project
-```
+Keycloak needs to be set up for the application to run properly, a keycloak user needs to be made, and we need some test data.
```

Comment:
a keycloak user (admin/admin)...

I don't know if the beginner would know that the admin account is the same name and password as the MDS user.

Reply:
Easy change, done.